recipes-bsp/u-boot/files/0027-arm-dts-add-the-m.2-support.patch

```diff
@@ -0,0 +1,145 @@
+From 09cc8b834d2015915d7176a73444d72db20a9852 Mon Sep 17 00:00:00 2001
+From: chao zeng <chao.zeng@siemens.com>
+Date: Fri, 7 Jan 2022 10:18:07 +0800
+Subject: [PATCH 27/28] arm:dts:add the m.2 support
+
+add the support for the m.2 board based on
+the iot2050 advanced board
```

Comment:
Here, some more words on the characteristics of this variant would be nice. And don't forget to update doc/board/siemens/iot2050.rst!

Reply:
The doc should at least mention the new variant, then stating that it build process wise just like pg2.